app/build.gradle

```diff
@@ -20,9 +24,24 @@ android {
 }
 
 dependencies {
+    def appcompat_version = "1.1.0"
+    def constraintlayout_version = "1.1.3"
+    def cardview_version = "1.0.0"
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "androidx.appcompat:appcompat:$appcompat_version"
```

Comment:
`appcompat_version` is only used in `androidx.appcompat:appcompat:$appcompat_version`, why we need to define a variable? Same for `constraintlayout_version` and `cardview_version`.

Reply:
Don't really know why I made variables for these